app/models/concerns/hyrax/admin_set_behavior.rb

```diff
@@ -32,6 +32,20 @@ def self.default_set?(id)
         id == DEFAULT_ID
       end
 
+      # Creates the default AdminSet and an associated PermissionTemplate with workflow
+      # rubocop:disable Lint/HandleExceptions
+      def self.create_default!
+        return if AdminSet.exists?(DEFAULT_ID)
+        AdminSet.create!(id: DEFAULT_ID, title: ['Default Admin Set']) do |_as|
+          PermissionTemplate.create!(admin_set_id: DEFAULT_ID)
+        end
+      rescue ActiveFedora::IllegalOperation
```

Comment:
@jcoyne I thought that too, but could have sworn spec failed without it... though that may have been a fluke. I'll remove and test again.